gateway/src/connectors/defira/defira.ts

```diff
@@ -13,38 +13,17 @@ import {
   ContractTransaction,
 } from '@ethersproject/contracts';
 import {
-  // TODO: use this instead of UniswapRouter
-  Router,
-  // TODO: use this instead of UniswapPair. Note that its constructor takes in an optional factory address
-  // parameter that SHOULD be populated since the default only works for mainnet
-  Pair,
+  Router as DefiraRouter,
+  Pair as DefiraPair,
   SwapParameters,
-  // TODO: use this instead of UniswapTrade
-  Trade,
+  Trade as DefiraTrade,
+  Fetcher as DefiraFetcher,
 } from '@zuzu-cat/defira-sdk';
 
 import {
-  // TODO: use this instead of UniswapRouter
-  Router as UniswapRouter,
-  // TODO: use @uniswap/sdk-core Percent instead of this one
-  Percent as UniswapPercent,
-  // TODO: use @uniswap/sdk-core Token instead of this one
-  Token as UniswapToken,
-  // TODO: use @uniswap/sdk-core CurrencyAmount instead of this one
-  TokenAmount as UniswapTokenAmount,
-  // TODO: use defira-sdk Pair instead of this one once defira-sdk Fetcher is hooked in
-  Pair as UniswapPair,
-  // TODO: use defira-sdk Fetcher instead of this one
-  Fetcher as UniswapFetcher,
-  // TODO: use defira-sdk Trade instead of this one
-  Trade as UniswapTrade,
-} from '@uniswap/sdk';
-import {
-  // TODO: use this instead of UniswapPercent
   Percent,
-  // TODO: use this instead of UniswapTokenAmount
-  CurrencyAmount,
-  // TODO: use this for some generic parameters
+  Token as DefiraToken,
```

Comment:
I was also saying no to these. None of the stuff from the core Uniswap library is Defira specific.